app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb

```diff
@@ -52,7 +52,7 @@ def deploy_script
             param :hypervisor_password, String, :desc => N_("Hypervisor password, required for all hypervisor types except for libvirt"), :required => false
             param :satellite_url, String, :desc => N_("Satellite server FQDN"), :required => true
             param :debug, :bool, :desc => N_("Enable debugging output")
-            param :proxy, String, :desc => N_('HTTP proxy that should be used for communication between the server on which virt-who is running and the hypervisors and virtualization managers.'), :required => false
+            param :http_proxy_id, Integer, :desc => N_('HTTP proxy that should be used for communication between the server on which virt-who is running and the hypervisors and virtualization managers.')
```

Comment:
We also need a test for this. There's probably an existing test that can be modified to also include this parameter

Reply:
Added the proxy to this guy: https://github.com/theforeman/foreman_virt_who_configure/pull/129/files#diff-cc724df397a3e6d6dbd12982dcee6395d916fa5c730be94195abf341848fdcd9R143

It looks like we already had the ID added here in the main config here:

https://github.com/theforeman/foreman_virt_who_configure/blob/63453dac3a4dd713bf49774cfd79d4eb15dc5c52/test/functional/api/v2/configs_controller_test.rb#L17